client/src/components/App/Content/InvestigationForm/TabManagement/PersonalInfoTab/PersonalInfoTabStyles.ts

```diff
@@ -125,6 +125,12 @@ const useStyles = makeStyles({
         height: '3.99vh',
         '@media screen and (max-height: 950px)': {
             height: '5.9vh'
+        },
+    },
+    otherTextField: {
+        width: '100%',
+        '&.MuiOutlinedInput-root': {
+            height: '42px'
```

Comment:
Why px and not vh?

Reply:
changed it to minHieghet so need fixed size that suits the other institution fields